pkg/apis/policies/v1/policyserver_webhook_test.go

```diff
@@ -16,14 +16,41 @@ package v1
 
 import (
 	"context"
+	"errors"
 	"testing"
 
+	"github.com/kubewarden/kubewarden-controller/internal/pkg/constants"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+func TestDefault(t *testing.T) {
+	policyServer := &PolicyServer{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "policy-server",
+		},
+		Spec: PolicyServerSpec{
+			Limits: corev1.ResourceList{
+				"cpu":    resource.MustParse("1Gi"),
```

Comment:
I suppose this is ok, but 1Gi CPU units sounds weird.

Reply:
fixed in https://github.com/kubewarden/kubewarden-controller/pull/708/commits/836340188d0af4b387a419455b6d618e1886d979